detailsPopup.js

```diff
@@ -0,0 +1,165 @@
+const projects = [
+  {
+    name: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'assets/Snapshoot Portfolio.svg',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: 'https://hjlk.jk/dfjkls',
+    linkToSource: 'https://gfsdf.hjlhs8h',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'assets/Snapshoot Portfolio-2.svg',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: 'https://hjlk.jk/dfjkls',
+    linkToSource: 'https://gfsdf.hjlhs8h',
+  },
+  {
+    name: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'assets/Snapshoot Portfolio-3.svg',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: 'https://hjlk.jk/dfjkls',
+    linkToSource: 'https://gfsdf.hjlhs8h',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'assets/Snapshoot Portfolio-4.svg',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: 'https://hjlk.jk/dfjkls',
+    linkToSource: 'https://gfsdf.hjlhs8h',
+  },
+];
+
+const classNames = ['work-one', 'work-two', 'work-three', 'work-four'];
+
+const skillInfoListItems = [
+  'CANOPY',
+  '<img src="assets/Counter.png" alt="bullet" />',
+  'Back End Dev',
+  '<img src="assets/Counter.png" alt="bullet" />',
+  '2015',
+];
+
+const languages = ['HTML', 'CSS', 'JavaScript'];
```

Comment:
Excellent job storing your project information in an array of objects just as [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), but let's ensure information about our project is stored inside the array of objects 😉 I can see you already have this stored in your array as `technologies` so there's no need to store it again. you can access the one you have created in the main object as this makes your code dry, neater, and shorter👍 Kindly ensure every information about your projects is accessed from the array of objects you have created for your projects so as to make your codebase maintainable and dry😉.